src/js/Alert.js

```diff
@@ -6,15 +6,11 @@ import Image from './Templates/Shared/Image'
 import StaticIcon from './Templates/Shared/StaticIcon'
 import store from './store.js'
 import UIController from './Controllers/UIController'
-import TTSController from './Controllers/TTSController'
-import { connect } from 'react-redux'
 
-import { updateResetPeriod, resetTimeout } from './actions';
 
-
-export const DEFAULT_RESET_TIMEOUT = 10000
+export const DEFAULT_RESET_TIMEOUT = 10000;
 const OUT_OF_BOUND_RESET_PERIOD = 1000;
-class Alert extends React.Component {
+export default class Alert extends React.Component {
```

Comment:
@LiliiaShlikhtLuxoft What for it's added `export default` ?

Reply:
@ValeriiMalkov because every class should be exported, but previous commit changed it and put it at the end of class (for props purpose). I returned it.